### PR TITLE
Add more instructions for postgres setup

### PR DESCRIPTION
**Pull request by lilyosah**
- Add some additional instructions to get DB set up. Eventually these commands will need to be automated by either a bash/bat script that runs an SQL file or by running a SQL file.
- Move instructions about development from Readme.md -> Contributing.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +20,25 @@ If there are any issues with these instructions or if anything is unclear, pleas
...
 
-5. [Install PostgreSQL](https://www.postgresql.org/download/)
+5. Set up the database:
+    - [Install PostgreSQL](https://www.postgresql.org/download/)
+    - (Windows) Set environment variables after install following [step 3 of this guide](https://medium.com/@aeadedoyin/getting-started-with-postgresql-on-windows-201906131300-ee75f066df78)
+    - Start server and start psql
+
+        ```bash
+        pg_ctl start
+        psql postgres
```
Comment (bcchin) [inline README.md]: I installed PostgreSQL with Homebrew so the server starts automatically for me and i only need to run`psql postgres` to start psql - not sure if you want to include this as a note in the README

File: setup.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE quill_db WITH 
+    ENCODING = 'UTF8' 
+    TEMPLATE = template0;
+CREATE USER quill_user WITH ENCRYPTED PASSWORD 'PASS_IN_ENV';
+-- Password shouldn't be hard-coded in here either if we are pushing it
+ALTER ROLE quill_user SET client_encoding TO 'utf8';
+ALTER ROLE quill_user SET default_transaction_isolation TO 'read committed';
+ALTER ROLE quill_user SET timezone TO 'UTC';
+GRANT ALL PRIVILEGES ON DATABASE quill_db TO postgres
```
Comment (bcchin) [inline setup.sql:9]: should be `quill_user` and not `postgres` since we just created that user with the roles

Comment (bcchin) [reply setup.sql:9]: also, could you add `\q` to this file so people know how to quit out of psql command line?

**Review by bcchin — CHANGES_REQUESTED** [review]

File: setup.sql (hunk 2 above)
Comment (lilyosah) [reply setup.sql:9]: Ah okay. Is quill_user like our admin then? OR a stand in for any user? My reasoning is we don't want them to be able to drop tables and stuff. And I'll add the note about `\q`

File: README.md (hunk 1 above)
Comment (lilyosah) [reply README.md]: Added!

File: setup.sql (hunk 2 above)
Comment (bcchin) [reply setup.sql:9]: your reasoning makes sense. as the commands are written right now, it's a stand-in for any user. i think we would have to make it a superuser in order for it to be our admin.

Comment (lilyosah) [reply setup.sql:9]: Ah okay, sounds good!

**Review by bcchin — APPROVED** [review]

**Pull request merged**